test/cli/commands_test.py

```diff
@@ -510,11 +510,16 @@ def test__cli__command_lint_serialize_multiple_files(serialize):
 
     if serialize == "json":
         result = json.loads(result.output)
+        assert len(result) == 2
     elif serialize == "yaml":
         result = yaml.safe_load(result.output)
+        assert len(result) == 2
+    elif serialize == "github-annotation":
+        result = json.loads(result.output)
```

Comment:
Let's add a unit test that asserts the full response?

Reply:
Done!